doc/releases/release-notes-2.6.rst

```diff
@@ -333,28 +331,30 @@ Drivers and Sensors
 
 * Counter
 
-* Crypto
+   * Added support for ESP32 Counter
 
 * DAC
 
-* Debug
+   * Added support for Microchip MCP4725
 
 * Disk
 
   * Added SDMMC support on STM32L4+
 
 * Display
 
+  * Added support for ST7735R
+
 * DMA
 
   * Added support on STM32G0 and STM32H7
 
 * EEPROM
```

Comment:
You can remove the EEPROM section as well. No major updates there for v2.6.0.

Reply:
looks like we added EEPROM simulator.